bower.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "jquery-ajax-unobtrusive",
-    "version": "3.2.4",
+    "version": "3.2.5",
```

Comment:
Yeah, I agree. We should only update versions when there is actual change to the .js file logic.